### ISSUE TITLE
Mysterious "CMD M105" being issued at interval

### ISSUE THREAD

**Issue by Mobyn** [arendst/Sonoff-Tasmota#2392]
NodeMCU running 5.12.0 as 18 Generic flashed via esptool
Relay 1 on GPIO 12
Relay 2 on GPIO 15

I'm getting a ghost command pop up on the console every 2 seconds

13:10:28 CMD: 
13:10:28 MQT: stat/Garage_Sonoff/RESULT = {"Command":"Unknown"}
13:10:28 CMD: M105
13:10:28 MQT: stat/Garage_Sonoff/RESULT = {"Command":"Unknown"}
13:10:30 CMD: 
13:10:30 MQT: stat/Garage_Sonoff/RESULT = {"Command":"Unknown"}
13:10:30 CMD: M105
13:10:30 MQT: stat/Garage_Sonoff/RESULT = {"Command":"Unknown"}

Not actually creating problems except when I'm trying use the console effectively. 

Amazing piece of code, thank you so much for building and supporting it!

**Comment by Jason2866** [comment]
Which Home Automation system do you use?

**Comment by Mobyn** [comment]
HASS. I fixed the issue by flashing again. It was obviously something I did w/ console commands

**Comment by Geitde** [comment]
M105 sounds like the temperature polling from a 3D Printer.

**Comment by Mobyn** [comment]
You are TOTALLY right! Odd that it's coming through the console of sonoff webserver. My other mqtt messages don't
